Jenkinsfile

```diff
@@ -12,34 +11,98 @@ def show_node_info() {
     """
 }
 
-DOCKER_IMAGE = "megatron-lm"
-CONTAINER_NAME = "megatron-lm-container"
-DOCKER_BUILD_ARGS = "--build-arg PYTORCH_ROCM_ARCH_OVERRIDE=gfx90a"
-DOCKER_RUN_ARGS = "-v \$(pwd):/workspace/Megatron-LM/output --workdir /workspace/Megatron-LM --entrypoint /workspace/Megatron-LM/run_unit_tests.sh"
+def clean_up_docker_containers() {
+    sh """
+    set +e
+    docker stop ${env.CONTAINER_NAME} || true
```

Comment:
Why do we need `|| true` here?

Reply:
@wenchenvincent 
"||true" is added so that it doesn't fail the pipeline even if this command fails ( e.g. when no container is found).
So I am running the cleanup before the test ( to make to cleanup the existing container). 
Then afterwards, to cleanup the containers.

It is debatable if need to cleanup before the tests. I just added to be sure, in case it is present for some reason.